src/helix-ui/styles/components/lists.less

```diff
@@ -1,24 +1,61 @@
 @import (reference) 'components/grid';
 
-hx-def + hx-def {
+dl {
+  display: block;
+  margin: 0;
+}
+
+dl > div {
+  display: flex;
+}
+
+div + div {
   margin-top: @grid-gutter-size; // 20px
 }
 
-hx-dt {
+dt {
   #grid.span(2);
   color: @gray-750;
+  display: block;
   flex-shrink: 1;
   order: 1;
   text-align: right;
 }
 
-hx-dd {
+dd {
   #grid.span(10);
+  display: block;
   flex-shrink: 1;
   margin-left: @grid-gutter-size; // 20px
   order: 2;
 }
 
+dl.hxVertical {
+  div {
```

Comment:
Needs qualified with direct child selector (`> div`) so that it doesn't match divs within `<dt>` or `<dd>` elements.